packages/storage-evm/README-energy-demo.md

```
@@ -0,0 +1,179 @@
+### Environment variables
+
+| Variable | Default | Description |
+|---|---|---|
+| `ENERGY_DEMO_COMMAND` | `loop` | `deploy` = deploy new community, `once` = run one batch, `loop` = run continuously, `reset` = call `emergencyReset()` |
+| `ENERGY_DEMO_LOOP_MS` | `45000` | Milliseconds between batches in loop mode |
+| `DEPLOY_FACTORY` | — | Set to `1` to force deploying a new factory (instead of reusing existing) |
+| `PRIVATE_KEY` | — | Deployer/admin wallet private key |
+
+### Demo community layout
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)